packages/block-library/src/latest-posts/edit.js

```diff
@@ -491,13 +490,13 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 									) }
 								</div>
 							) }
-							<a href={ post.link } rel="noreferrer noopener">
-								{ titleTrimmed ? (
-									<RawHTML>{ titleTrimmed }</RawHTML>
-								) : (
-									__( '(no title)' )
-								) }
-							</a>
+							<a
+								href={ post.link }
+								rel="noreferrer noopener"
+								dangerouslySetInnerHTML={ {
+									__html: titleTrimmed || __( '(no title)' ),
```

Comment:
The translation should remain sanitized. It’s only the tittle that’s is already sanitized on the server so we consider it’s content as safe.

Reply:
Updated in: https://github.com/WordPress/gutenberg/pull/38527/commits/d16e2bbce16bc229a21c8026268efc7e378e964b